gamepage.py

```
@@ -59,18 +61,28 @@ def setup_ui(self):
                 Text(f"Link2: {self.game.link2}", style="body2"),
                 Text(f"Hide: {self.game.hide}", style="body2"),
                 Text(f"Created Date: {self.game.createddate}", style="body2"),
-                Text(f"Metadata: {self.game.metadata}", style="body2"),
+                # Additional metadata fields
+                Text(f"Type: {self.game.metadata.get('type', 'N/A')}", style="body2"),
```

Ruff (0.8.2)

`Text` may be undefined, or defined from star imports

(F405)